src/main/java/no/nav/familie/integrasjoner/client/soap/PersonSoapClient.kt

```diff
@@ -125,6 +127,8 @@ class PersonSoapClient(private val port: PersonV3) : AbstractSoapClient("personV
     }
 
     private fun sjekkConnectionReset(e: Exception): Boolean {
-        return e.message != null && e.message!!.contains("Connection reset")
+        val sw = StringWriter()
+        e.printStackTrace(PrintWriter(sw))
```

Comment:
Er det ikke mulig å bruke `NestedExceptionUtils.getMostSpecificCause` her? Pleier vel å være root cause som har connection reset?

Reply:
Antaglig.  Vi er ikke sikker på om den er mest spesifikk. Dette vil fungere for en ting som skaper mye støy.